spec/models/sponsor_spec.rb

```diff
@@ -328,6 +328,24 @@
       expect(request_fulfilled_sponsor.eligible_for_sponsorship?).to eq false
     end
 
+    describe '#sponsorship_changed!', :mytest => true do
+
+      before(:each) do
+        allow(active_sponsor).to receive(:update_request_fulfilled!)
+        allow(active_sponsor).to receive(:update_active_sponsorship_count!)
+        active_sponsor.sponsorship_changed!
+      end
+
+      it 'should call #update_request_fulfilled!' do
+        expect(active_sponsor).to have_received(:update_request_fulfilled!)
+      end
+      
+      it 'should call #update_active_sponsorship_count!' do
+        expect(active_sponsor).to have_received(:update_active_sponsorship_count!)
+      end
```

Comment:
Without stubs:

```
Sponsor methods #sponsorship_changed! #sponsorship_changed! should call update methods
     Failure/Error: expect(active_sponsor).to have_received(:update_request_fulfilled!)
       #<Sponsor:0xfa95560> expected to have received update_request_fulfilled!, but that object is not a spy or method has not been stubbed.
```

Your second suggestion works though:

```ruby
      specify '#sponsorship_changed! should call update methods' do
        expect(active_sponsor).to receive(:update_request_fulfilled!)
        expect(active_sponsor).to receive(:update_active_sponsorship_count!)
        active_sponsor.sponsorship_changed!
      end
```
I am pretty sure though I tried something similar to `expect(active_sponsor).to receive(:update_request_fulfilled!)` (although not as a first line) and got rspec complaining that it was expecting two arguments (to receive...with...) instead of just one. Very confusing.